aws-sns-subscription/src/main/java/software/amazon/sns/subscription/Translator.java

```diff
@@ -33,16 +38,17 @@ static GetSubscriptionAttributesRequest translateToReadRequest(final ResourceMod
   static ResourceModel translateFromReadResponse(final GetSubscriptionAttributesResponse getSubscriptionAttributesResponse) {
     final Map<String, String> attributes = getSubscriptionAttributesResponse.attributes();
 
-    Boolean rawMessageDelivery = attributes.get(Definitions.rawMessageDelivery) != null ? Boolean.valueOf(attributes.get(Definitions.rawMessageDelivery)) : null;
-    return ResourceModel.builder().subscriptionArn(attributes.get(Definitions.subscriptionArn))
+    return ResourceModel.builder()
+                            .subscriptionArn(attributes.get(Definitions.subscriptionArn))
                             .topicArn(attributes.get(Definitions.topicArn))
                             .endpoint(attributes.get(Definitions.endpoint))
                             .protocol(attributes.get(Definitions.protocol))
                             .filterPolicy(attributes.get(Definitions.filterPolicy) != null ? SnsSubscriptionUtils.convertToJson(attributes.get(Definitions.filterPolicy)) : null)
                             .redrivePolicy(attributes.get(Definitions.redrivePolicy) != null ? SnsSubscriptionUtils.convertToJson(attributes.get(Definitions.redrivePolicy)) : null)
                             .deliveryPolicy(attributes.get(Definitions.deliveryPolicy) != null ? SnsSubscriptionUtils.convertToJson(attributes.get(Definitions.deliveryPolicy)) : null)
-                            .rawMessageDelivery(rawMessageDelivery)
+                            .rawMessageDelivery(attributes.get(Definitions.rawMessageDelivery) != null ? Boolean.valueOf(attributes.get(Definitions.rawMessageDelivery)) : null)
```

Comment:
will this always be null if it isn't entered? could it be empty? asking if it would be better to do a NotNullOrEmpty() check instead of only checking for null